src/test/java/org/reactome/addlinks/test/EnsemblBioMartRetrieverIT.java

```diff
@@ -17,24 +17,25 @@
 
     private static final String TEST_DIRECTORY = "/tmp/test_ensembl_biomart_mapping_service/";
     private static final String EXPECTED_ERROR_MESSAGE = "BioMartQueryException: BioMart query failed with message: Query ERROR";
+    private static final String BIOMART_URL = "http://www.ensembl.org/biomart/martservice?";
```

Comment:
I think EnsEMBL BioMart supports HTTPS, so you could use that in place of HTTP here.

Reply:
Yep that worked, I've pushed that change